### ISSUE TITLE
Error: External Program Execution Failed

### ISSUE THREAD

**Issue by adithyaGR** [terraform-google-modules/terraform-google-project-factory#772]
### TL;DR

Im trying to run the code as mentioned in the documentation https://github.com/terraform-google-modules/terraform-google-gcloud/ but im unable to run it locally and I get the error as mentioned here 

### Expected behavior

The code should tun without any issues

### Observed behavior

$ terraform plan
module.gcloud_test.data.external.env_override[0]: Reading...
╷
│ Error: External Program Execution Failed
│
│   with module.gcloud_test.data.external.env_override[0],
│   on terraform-google-gcloud\main.tf line 73, in data "external" "env_override":
│   73:   program = ["${path.module}/scripts/check_env.sh"]
│
│ The data source received an unexpected error while attempting to execute
│ the program.
│
│ Program: terraform-google-gcloud/scripts/check_env.sh
│ Error: fork/exec terraform-google-gcloud/scripts/check_env.sh: %1 is not a

### Terraform Configuration

```hcl
module "gcloud_test" {
    source = "./terraform-google-gcloud"
    create_cmd_entrypoint  = "gcloud"
    create_cmd_body        = "version"
}
```

### Terraform Version

```sh
$ terraform -version
Terraform v1.3.6
on windows_amd64
+ provider registry.terraform.io/hashicorp/external v2.2.3
+ provider registry.terraform.io/hashicorp/google v4.44.1
+ provider registry.terraform.io/hashicorp/null v3.2.1
+ provider registry.terraform.io/hashicorp/random v3.4.3
```

### Additional information

Both TF version v1.3.6 and 1.2.0 throws the error as mentioned here

**Comment by adithyaGR** [comment]
│ Error: fork/exec terraform-google-gcloud/scripts/check_env.sh: %1 is not a
│ valid Win32 application.

**Comment by bharathkkb** [comment]
Thanks for the report @adithyaGR 
This seems related to https://github.com/terraform-google-modules/terraform-google-gcloud/issues/84 and not this module. I have reopened that issue but I suspect there is no easy way for us to support this.
